integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -609,10 +613,33 @@ void testLogHomeChangeEvents() {
     }
 
     //verify the logHome change causes the domain roll events to be logged
+    verifyDomainRollAndPodCycleEvents(timestamp);
+  }
+
+  private void verifyDomainRollAndPodCycleEvents(OffsetDateTime timestamp) {
     logger.info("verify domain roll starting/pod cycle starting events are logged");
     checkEvent(opNamespace, domainNamespace1, domainUid, DOMAIN_ROLL_STARTING, "Normal", timestamp);
     checkEvent(opNamespace, domainNamespace1, domainUid, POD_CYCLE_STARTING, "Normal", timestamp);
     checkEvent(opNamespace, domainNamespace1, domainUid, DOMAIN_ROLL_COMPLETED, "Normal", timestamp);
+    verifyDomainStatusConditionTypeDoesNotExist(domainUid, domainNamespace1, "Rolling");
+  }
+
+  @NotNull
+  private Domain getAndValidateInitialDomain(String domainNamespace) {
```

Comment:
I'm going to approve to try and get this in tonight; however, if this method is a copy of what is in ItPodsRestart then please move it to a common place.